packages/contracts/deployments/engine/V3/factory-and-implementations/v3.2/V3_Engine_Implementation_v3.2.4.md

```diff
@@ -26,4 +26,5 @@ Deploys to address: `0x00000000559cA3F3f1279C0ec121c302ed010457`
 
 - sepolia: https://sepolia.etherscan.io/tx/0x06a52386e4856c52a610b382d833480b8ccaef758303be0e758751aed2a87a0e
 - arbitrum: https://arbiscan.io/tx/0x09ce4be9e898b62e9951db3d77b8579531f400d183a5cb85bda219438c52abc1
+- base: TODO
```

Comment:
Are we waiting on basescan for this one?

Reply:
oops, fixed in [884cb29](https://github.com/ArtBlocks/artblocks-contracts/pull/1606/commits/884cb297200dad3d28984d08bcc2fe1858b71cee)